util/pyutil/big5.py

```diff
@@ -3,7 +3,7 @@
 from __future__ import unicode_literals
 from builtins import bytes, chr
 
-import big5_tbl
+from . import big5_tbl
```

Comment:
please rephrase your commit message as

Use explicit relative import for python3